src/utils/UserContext.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+type User = {
+    id: number
+    name: string
+    description: string
+  }
+
+const usersData = React.createContext<User[]>([]);
```

Comment:
By convention, context const must be `PascalCase`, and should look like `UserContext` and not data.